.travis.yml

```diff
@@ -17,11 +38,11 @@ matrix:
   - os: linux
     compiler: clang
     env: Q_OR_C_MAKE=cmake
-
```

Comment:
As I said, this version was the first test. I opened the PR to get travis to build with the new configuration. I didn't get to have a look at the whole process yet.